librz/arch/opcodes/meson.build

```diff
@@ -18,6 +18,8 @@ sdb_opcodes_files = [
   'ppc',
   'propeller',
   'riscv',
+  'rx',
+  'rl78',
```

Comment:
i don't see the `rl78.sdb.txt` in the PR

Reply:
It exists already but wasn't added in the buildsystem for some reason.